### ISSUE TITLE
Replace operation stucked and errors Failed to handle STREAM_MUTATION_FRAGMENTS Operation timed out for system.view_build_status_v2 (while cleaning up after std::runtime_error (Dangling queue_reader_handle_v2

### ISSUE THREAD

**Issue by aleksbykov** [scylladb/scylladb#21162]
## Packages

Scylla version: `6.3.0~dev-20241006.960aa38cf32b` with build-id `d9c1c78fef9b7b6f55a36909aa3cf250f3dbe4a7`

Kernel Version: `6.8.0-1016-aws`

## Issue description

- [x] This issue is a regression.
- [ ] It is unknown if this issue is a regression.

Cluster configured with 2 Datacenters. 2 nemesis ran in parallel. In 1st dc start CreateINdex nemesis in second DC terminate and replace.
Node5 was termianted and then node9 was bootstrapping with replace for node 5.
The cluster was configured with tablets enabled.
Both operations, creating the index and replacing it, took over 7 hours and were eventually aborted due to a timeout. It seems that the tablet streaming got stuck at some point.

In logs tablets rebuilding was started:
```
2024-10-07T12:55:15.673+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9     !INFO | scylla[6504]:  [shard 0:strm] range_streamer - Tablet rebuild starts, nr_ranges_remaining=5
2024-10-07T12:55:15.673+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9     !INFO | scylla[6504]:  [shard 0:strm] range_streamer - Tablet rebuild with [10.4.4.156, 10.4.3.232, 10.3.6.202, 10.4.3.125, 10.3.3.89] for keyspace=keyspace1 started, nodes_to_stream=5
2024-10-07T12:55:15.673+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9     !INFO | scylla[6504]:  [shard 0:strm] range_streamer - Tablet rebuild with 10.4.4.156 for keyspace=keyspace1, streaming [0, 1) out of 1 ranges

```
at some point next errors appeared and after that 
```
2024-10-07T12:55:15.698+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9     !INFO | scylla[6504]:  [shard 0:strm] stream_session - [Stream #65f5db31-84ab-11ef-b5fb-80f7bf592247] Executing streaming plan for Tablet rebuild-keyspace1-index-0 with peers={10.3.3.89}, master
2024-10-07T12:55:25.816+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9     !INFO | systemd[1506]: launchpadlib-cache-clean.service - Clean up old files in the Launchpadlib cache was skipped because of an unmet condition check (ConditionPathExists=/home/scyllaadm/.launchpadlib/api.launchpad.net/cache).
2024-10-07T12:55:26.307+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9     !INFO | scylla[6504]:  [shard 0:strm] reader_concurrency_semaphore - Semaphore streaming with 10/10 count and 305897/175951052 memory resources: timed out, dumping permit diagnostics:
Trigger permit: count=0, memory=0, table=system.view_build_status_v2, operation=data-query, state=waiting_for_admission
Identified bottleneck(s): disk

permits	count	memory	table/operation/state
7	7	222K	keyspace1.sec_ind_c3_index/stream-session/active
3	3	77K	keyspace1.standard2/stream-session/active
4	0	0B	keyspace1.standard2/stream-session/waiting_for_admission
20	0	0B	system.view_build_status_v2/data-query/waiting_for_admission

34	10	299K	total

Stats:
permit_based_evictions: 0
time_based_evictions: 0
inactive_reads: 0
total_successful_reads: 79
total_failed_reads: 0
total_reads_shed_due_to_overload: 0
total_reads_killed_due_to_kill_limit: 0
reads_admitted: 7910
reads_enqueued_for_admission: 40
reads_enqueued_for_memory: 0
reads_admitted_immediately: 7894
reads_queued_because_ready_list: 15
reads_queued_because_need_cpu_permits: 1
reads_queued_because_memory_resources: 24
reads_queued_because_count_resources: 0
reads_queued_with_eviction: 0
total_permits: 7934
current_permits: 34
need_cpu_permits: 0
awaits_permits: 0
disk_reads: 0
sstables_read: 0
2024-10-07T12:55:26.307+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9      !ERR | scylla[6504]:  [shard 5:strm] stream_session - [Stream #65f5b41a-84ab-11ef-b5fb-80f7bf592247] Failed to handle STREAM_MUTATION_FRAGMENTS (receive and distribute phase) for ks=keyspace1, cf=standard2, peer=10.4.3.232: seastar::nested_exception: exceptions::read_timeout_exception (Operation timed out for system.view_build_status_v2 - received only 0 responses from 1 CL=LOCAL_ONE.) (while cleaning up after exceptions::read_timeout_exception (Operation timed out for system.view_build_status_v2 - received only 0 responses from 1 CL=LOCAL_ONE.))
2024-10-07T12:55:26.307+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9      !ERR | scylla[6504]:  [shard 3:strm] stream_session - [Stream #65f5b412-84ab-11ef-b5fb-80f7bf592247] Failed to handle STREAM_MUTATION_FRAGMENTS (receive and distribute phase) for ks=keyspace1, cf=standard2, peer=10.4.3.125: seastar::nested_exception: exceptions::read_timeout_exception (Operation timed out for system.view_build_status_v2 - received only 0 responses from 1 CL=LOCAL_ONE.) (while cleaning up after std::runtime_error (Dangling queue_reader_handle_v2))
2024-10-07T12:55:26.307+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9      !ERR | scylla[6504]:  [shard 2:strm] stream_session - [Stream #65f5b410-84ab-11ef-b5fb-80f7bf592247] Failed to handle STREAM_MUTATION_FRAGMENTS (receive and distribute phase) for ks=keyspace1, cf=standard2, peer=10.4.3.232: seastar::nested_exception: exceptions::read_timeout_exception (Operation timed out for system.view_build_status_v2 - received only 0 responses from 1 CL=LOCAL_ONE.) (while cleaning up after std::runtime_error (Dangling queue_reader_handle_v2))
2024-10-07T12:55:26.307+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9      !ERR | scylla[6504]:  [shard 2:strm] stream_session - [Stream #65f5db2e-84ab-11ef-b5fb-80f7bf592247] Failed to handle STREAM_MUTATION_FRAGMENTS (receive and distribute phase) for ks=keyspace1, cf=standard2, peer=10.4.3.232: seastar::nested_exception: exceptions::read_timeout_exception (Operation timed out for system.view_build_status_v2 - received only 0 responses from 1 CL=LOCAL_ONE.) (while cleaning up after std::runtime_error (Dangling queue_reader_handle_v2))
2024-10-07T12:55:26.307+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9      !ERR | scylla[6504]:  [shard 6:strm] stream_session - [Stream #65f5b41e-84ab-11ef-b5fb-80f7bf592247] Failed to handle STREAM_MUTATION_FRAGMENTS (receive and distribute phase) for ks=keyspace1, cf=standard2, peer=10.4.3.125: seastar::nested_exception: exceptions::read_timeout_exception (Operation timed out for system.view_build_status_v2 - received only 0 responses from 1 CL=LOCAL_ONE.) (while cleaning up after std::runtime_error (Dangling queue_reader_handle_v2))
2024-10-07T12:55:26.307+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9      !ERR | scylla[6504]:  [shard 6:strm] stream_session - [Stream #65f5b41f-84ab-11ef-b5fb-80f7bf592247] Failed to handle STREAM_MUTATION_FRAGMENTS (receive and distribute phase) for ks=keyspace1, cf=standard2, peer=10.4.4.156: seastar::nested_exception: exceptions::read_timeout_exception (Operation timed out for system.view_build_status_v2 - received only 0 responses from 1 CL=LOCAL_ONE.) (while cleaning up after std::runtime_error (Dangling queue_reader_handle_v2))
2024-10-07T12:55:26.307+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9      !ERR | scylla[6504]:  [shard 6:strm] stream_session - [Stream #65f5db20-84ab-11ef-b5fb-80f7bf592247] Failed to handle STREAM_MUTATION_FRAGMENTS (receive and distribute phase) for ks=keyspace1, cf=standard2, peer=10.4.3.232: seastar::nested_exception: exceptions::read_timeout_exception (Operation timed out for system.view_build_status_v2 - received only 0 responses from 1 CL=LOCAL_ONE.) (while cleaning up after std::runtime_error (Dangling queue_reader_handle_v2))
2024-10-07T12:55:26.317+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9      !ERR | scylla[6504]:  [shard 5:strm] stream_session - [Stream #65f5b41b-84ab-11ef-b5fb-80f7bf592247] Failed to handle STREAM_MUTATION_FRAGMENTS (receive and distribute phase) for ks=keyspace1, cf=standard2, peer=10.4.4.156: seastar::nested_exception: exceptions::read_timeout_exception (Operation timed out for system.view_build_status_v2 - received only 0 responses from 1 CL=LOCAL_ONE.) (while cleaning up after exceptions::read_timeout_exception (Operation timed out for system.view_build_status_v2 - received only 0 responses from 1 CL=LOCAL_ONE.))
2024-10-07T12:55:26.317+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9      !ERR | scylla[6504]:  [shard 6:strm] stream_session - [Stream #65f5db21-84ab-11ef-b5fb-80f7bf592247] Failed to handle STREAM_MUTATION_FRAGMENTS (receive and distribute phase) for ks=keyspace1, cf=standard2, peer=10.4.4.118: seastar::nested_exception: exceptions::read_timeout_exception (Operation timed out for system.view_build_status_v2 - received only 0 responses from 1 CL=LOCAL_ONE.) (while cleaning up after std::runtime_error (Dangling queue_reader_handle_v2))
2024-10-07T12:55:26.317+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9      !ERR | scylla[6504]:  [shard 2:strm] stream_session - [Stream #65f5b411-84ab-11ef-b5fb-80f7bf592247] Failed to handle STREAM_MUTATION_FRAGMENTS (receive and distribute phase) for ks=keyspace1, cf=standard2, peer=10.4.4.118: seastar::nested_exception: exceptions::read_timeout_exception (Operation timed out for system.view_build_status_v2 - received only 0 responses from 1 CL=LOCAL_ONE.) (while cleaning up after std::runtime_error (Dangling queue_reader_handle_v2))

```
the latest messages:
```
2024-10-07T13:02:46.066+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9     !INFO | scylla[6504]:  [shard 0:strm] range_streamer - Finished 5 out of 5 ranges for rebuild, finished percentage=1
2024-10-07T13:02:46.066+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9     !INFO | scylla[6504]:  [shard 0:strm] range_streamer - Tablet rebuild with 10.4.4.118 for keyspace=keyspace1 succeeded, took 161.34232 seconds
2024-10-07T13:02:46.066+00:00 parallel-topology-schema-changes-mu-db-node-70dd8179-9     !INFO | scylla[6504]:  [shard 0:strm] range_streamer - Tablet rebuild succeeded, took 161 seconds, nr_ranges_remaining=0
```

after that only compaction and bloom filter rebuilding messages appeared. 
And both nemesis waiting operation finished about 7 hours.

## Impact

Replace operation was not finished and took too much time 

## How frequently does it reproduce?

*Describe the frequency with how this issue can be reproduced.*

## Installation details

Cluster size: 8 nodes (i3en.2xlarge)

Scylla Nodes used in this run:
- parallel-topology-schema-changes-mu-db-node-70dd8179-9 (13.40.2.213 | 10.3.2.143) (shards: 7)
- parallel-topology-schema-changes-mu-db-node-70dd8179-8 (13.42.31.77 | 10.3.6.202) (shards: 7)
- parallel-topology-schema-changes-mu-db-node-70dd8179-7 (18.170.87.112 | 10.3.4.67) (shards: 7)
- parallel-topology-schema-changes-mu-db-node-70dd8179-6 (18.169.166.186 | 10.3.3.89) (shards: 7)
- parallel-topology-schema-changes-mu-db-node-70dd8179-5 (18.133.240.223 | 10.3.3.119) (shards: 7)
- parallel-topology-schema-changes-mu-db-node-70dd8179-4 (18.201.26.143 | 10.4.4.118) (shards: 7)
- parallel-topology-schema-changes-mu-db-node-70dd8179-3 (3.252.227.26 | 10.4.4.156) (shards: 7)
- parallel-topology-schema-changes-mu-db-node-70dd8179-2 (34.241.121.249 | 10.4.3.232) (shards: 7)
- parallel-topology-schema-changes-mu-db-node-70dd8179-1 (63.35.223.123 | 10.4.3.125) (shards: 7)

OS / Image: `ami-0608a992b92a02fa8 ami-077d3c43999a08e8d` (aws: undefined_region)

Test: `longevity-multidc-schema-topology-changes-12h-test`
Test id: `70dd8179-4c5b-46a2-9c44-9f3e40d904c0`
Test name: `scylla-staging/abykov/longevity-multidc-schema-topology-changes-12h-test`
Test method: `longevity_test.LongevityTest.test_custom_time`
Test config file(s):

- [longevity-multidc-parallel-topology-schema-changes-12h.yaml](https://github.com/scylladb/scylla-cluster-tests/blob/c063854ea2232b3fbd7b4c246d0197ccc3e7a046/test-cases/longevity/longevity-multidc-parallel-topology-schema-changes-12h.yaml)

<details>
<summary>
Logs and commands
</summary>

- Restore Monitor Stack command: `$ hydra investigate show-monitor 70dd8179-4c5b-46a2-9c44-9f3e40d904c0`
- Restore monitor on AWS instance using [Jenkins job](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=70dd8179-4c5b-46a2-9c44-9f3e40d904c0)
- Show all stored logs command: `$ hydra investigate show-logs 70dd8179-4c5b-46a2-9c44-9f3e40d904c0`

## Logs:
- **db-cluster-70dd8179.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/70dd8179-4c5b-46a2-9c44-9f3e40d904c0/20241007_204900/db-cluster-70dd8179.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/70dd8179-4c5b-46a2-9c44-9f3e40d904c0/20241007_204900/db-cluster-70dd8179.tar.gz)
- **sct-runner-events-70dd8179.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/70dd8179-4c5b-46a2-9c44-9f3e40d904c0/20241007_204900/sct-runner-events-70dd8179.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/70dd8179-4c5b-46a2-9c44-9f3e40d904c0/20241007_204900/sct-runner-events-70dd8179.tar.gz)
- **sct-70dd8179.log.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/70dd8179-4c5b-46a2-9c44-9f3e40d904c0/20241007_204900/sct-70dd8179.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/70dd8179-4c5b-46a2-9c44-9f3e40d904c0/20241007_204900/sct-70dd8179.log.tar.gz)
- **loader-set-70dd8179.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/70dd8179-4c5b-46a2-9c44-9f3e40d904c0/20241007_204900/loader-set-70dd8179.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/70dd8179-4c5b-46a2-9c44-9f3e40d904c0/20241007_204900/loader-set-70dd8179.tar.gz)
- **monitor-set-70dd8179.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/70dd8179-4c5b-46a2-9c44-9f3e40d904c0/20241007_204900/monitor-set-70dd8179.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/70dd8179-4c5b-46a2-9c44-9f3e40d904c0/20241007_204900/monitor-set-70dd8179.tar.gz)

[Jenkins job URL](https://jenkins.scylladb.com/job/scylla-staging/job/abykov/job/longevity-multidc-schema-topology-changes-12h-test/37/)
[Argus](https://argus.scylladb.com/test/b8c39260-a5e5-4e1b-8c73-ad922d4e6648/runs?additionalRuns[]=70dd8179-4c5b-46a2-9c44-9f3e40d904c0)
</details>

**Comment by aleksbykov** [comment]
The similar happened with but with segmentation fault and coredump

## Packages

Scylla version: `6.2.0~rc2-20241002.93700ff5d1ce` with build-id `57e6a907e0a03a185c4145bb8c7d6316b519b14c`

Kernel Version: `6.8.0-1016-aws`

## Installation details

Cluster size: 8 nodes (i3en.2xlarge)

Scylla Nodes used in this run:
- parallel-topology-schema-changes-mu-db-node-97087646-9 (54.229.119.88 | 10.4.0.193) (shards: 7)
- parallel-topology-schema-changes-mu-db-node-97087646-8 (13.40.214.189 | 10.3.4.178) (shards: 7)
- parallel-topology-schema-changes-mu-db-node-97087646-7 (13.41.202.28 | 10.3.6.254) (shards: 7)
- parallel-topology-schema-changes-mu-db-node-97087646-6 (13.40.3.27 | 10.3.1.173) (shards: 7)
- parallel-topology-schema-changes-mu-db-node-97087646-5 (3.8.171.82 | 10.3.1.127) (shards: 7)
- parallel-topology-schema-changes-mu-db-node-97087646-4 (3.250.183.163 | 10.4.7.197) (shards: 7)
- parallel-topology-schema-changes-mu-db-node-97087646-3 (34.254.253.182 | 10.4.4.123) (shards: 7)
- parallel-topology-schema-changes-mu-db-node-97087646-2 (108.129.165.3 | 10.4.0.233) (shards: 7)
- parallel-topology-schema-changes-mu-db-node-97087646-1 (34.253.92.183 | 10.4.2.51) (shards: 7)


OS / Image: `ami-04b7350d2c73a8b6d ami-0e2385be763c766a2` (aws: undefined_region)

Test: `longevity-multidc-schema-topology-changes-12h-test`
Test id: `97087646-fc82-4a56-a750-cf440126743d`
Test name: `scylla-staging/abykov/longevity-multidc-schema-topology-changes-12h-test`
Test method: `longevity_test.LongevityTest.test_custom_time`
Test config file(s):

- [longevity-multidc-parallel-topology-schema-changes-12h.yaml](https://github.com/scylladb/scylla-cluster-tests/blob/c063854ea2232b3fbd7b4c246d0197ccc3e7a046/test-cases/longevity/longevity-multidc-parallel-topology-schema-changes-12h.yaml)


<details>
<summary>
Logs and commands
</summary>


- Restore Monitor Stack command: `$ hydra investigate show-monitor 97087646-fc82-4a56-a750-cf440126743d`
- Restore monitor on AWS instance using [Jenkins job](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=97087646-fc82-4a56-a750-cf440126743d)
- Show all stored logs command: `$ hydra investigate show-logs 97087646-fc82-4a56-a750-cf440126743d`


## Logs:
- **core.scylla-parallel-topology-schema-changes-mu-db-node-97087646-9-2024-10-09_06-45-45.gz** - [https://storage.cloud.google.com/upload.scylladb.com/core.scylla.106.0a7949c18b4c43ec952edb26c9255c85.6512.1728455961000000./core.scylla.106.0a7949c18b4c43ec952edb26c9255c85.6512.1728455961000000.zst](https://storage.cloud.google.com/upload.scylladb.com/core.scylla.106.0a7949c18b4c43ec952edb26c9255c85.6512.1728455961000000./core.scylla.106.0a7949c18b4c43ec952edb26c9255c85.6512.1728455961000000.zst)
- **db-cluster-97087646.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/97087646-fc82-4a56-a750-cf440126743d/20241009_064923/db-cluster-97087646.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/97087646-fc82-4a56-a750-cf440126743d/20241009_064923/db-cluster-97087646.tar.gz)
- **sct-runner-events-97087646.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/97087646-fc82-4a56-a750-cf440126743d/20241009_064923/sct-runner-events-97087646.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/97087646-fc82-4a56-a750-cf440126743d/20241009_064923/sct-runner-events-97087646.tar.gz)
- **sct-97087646.log.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/97087646-fc82-4a56-a750-cf440126743d/20241009_064923/sct-97087646.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/97087646-fc82-4a56-a750-cf440126743d/20241009_064923/sct-97087646.log.tar.gz)
- **loader-set-97087646.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/97087646-fc82-4a56-a750-cf440126743d/20241009_064923/loader-set-97087646.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/97087646-fc82-4a56-a750-cf440126743d/20241009_064923/loader-set-97087646.tar.gz)
- **monitor-set-97087646.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/97087646-fc82-4a56-a750-cf440126743d/20241009_064923/monitor-set-97087646.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/97087646-fc82-4a56-a750-cf440126743d/20241009_064923/monitor-set-97087646.tar.gz)


[Jenkins job URL](https://jenkins.scylladb.com/job/scylla-staging/job/abykov/job/longevity-multidc-schema-topology-changes-12h-test/38/)
[Argus](https://argus.scylladb.com/test/b8c39260-a5e5-4e1b-8c73-ad922d4e6648/runs?additionalRuns[]=97087646-fc82-4a56-a750-cf440126743d)
</details>

```
2024-10-09T06:39:17.816+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9      !ERR | scylla[6512]:  [shard 2:strm] stream_session - [Stream #2f26bbf3-8609-11ef-ad75-39d2b6f3c536] Failed to handle STREAM_MUTATION_FRAGMENTS (receive and distribute phase) for ks=keyspace1, cf=standard1, peer=10.3.1.173: seastar::nested_exception: exceptions::read_timeout_exception (Operation timed out for system.view_build_status_v2 - received only 0 responses from 1 CL=LOCAL_ONE.) (while cleaning up after exceptions::read_timeout_exception (Operation timed out for system.view_build_status_v2 - received only 0 responses from 1 CL=LOCAL_ONE.))
2024-10-09T06:39:20.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | sshd[6691]: Accepted publickey for scyllaadm from 10.4.3.177 port 58836 ssh2: ED25519 SHA256:lqOFBZlL7juprDUVnOdbVILawZvWMcy7EcPFe6UrKAc
2024-10-09T06:39:20.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | sshd[6691]: pam_unix(sshd:session): session opened for user scyllaadm(uid=1000) by scyllaadm(uid=0)
2024-10-09T06:39:20.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | systemd-logind[602]: New session 17 of user scyllaadm.
2024-10-09T06:39:20.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | systemd[1]: Started session-17.scope - Session 17 of User scyllaadm.
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:  [shard 0:strm] stream_session - [Stream #1bf2a851-8609-11ef-ad75-39d2b6f3c536] Write to sstable for ks=keyspace1, cf=standard2, estimated_partitions=187776, received_partitions=187281
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9  !WARNING | scylla[6512]:  [shard 0:strm] token_metadata - topology version 64 held for 3.197 [s] past expiry, released at: 0x615fb8e 0x61601a0 0x61604a8 0x3d55d1a 0x3d48db6 0x3ca7c1e 0x3ca90f8 0x30bbb91 0x30ad178 0x31057bc 0x2bde99a 0x5c6276f 0x5c63cea 0x5c64ec7 0x5c64278 0x5bf43e3 0x5bf3743 0x13c75a5 0x13c8f60 0x13c59c3 /opt/scylladb/libreloc/libc.so.6+0x2a087 /opt/scylladb/libreloc/libc.so.6+0x2a14a 0x13c3044
   --------
   seastar::internal::coroutine_traits_base<boost::outcome_v2::basic_result<service::storage_proxy_coordinator_query_result, utils::exception_container<exceptions::mutation_write_timeout_exception, exceptions::read_timeout_exception, exceptions::read_failure_exception, exceptions::rate_limit_exception, exceptions::overloaded_exception>, utils::exception_container_throw_policy> >::promise_type
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]: Segmentation fault on shard 0, in scheduling group streaming.
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]: Backtrace:
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   0x5c944c4
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   0x5c5262b
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   0x5c88b7c
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   /opt/scylladb/libreloc/libc.so.6+0x40cff
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   0x3dae2b3
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   0x3d49c56
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   0x39e0bac
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   0x5c6276f
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   0x5c63cea
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   0x5c64ec7
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   0x5c64278
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   0x5bf43e3
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   0x5bf3743
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   0x13c75a5
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   0x13c8f60
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   0x13c59c3
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   /opt/scylladb/libreloc/libc.so.6+0x2a087
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   /opt/scylladb/libreloc/libc.so.6+0x2a14a
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:   0x13c3044

```

                          


Decoded backtrace of segmentation:
```
2024-10-09T06:39:21.565+00:00 parallel-topology-schema-changes-mu-db-node-97087646-9     !INFO | scylla[6512]:
[Backtrace #0]
void seastar::backtrace<seastar::backtrace_buffer::append_backtrace()::{lambda(seastar::frame)#1}>(seastar::backtrace_buffer::append_backtrace()::{lambda(seastar::frame)#1}&&) at ./build/release/seastar/./seastar/include/seastar/util/backtrace.hh:68
seastar::print_with_backtrace(seastar::backtrace_buffer&, bool) at ./build/release/seastar/./build/release/seastar/./seastar/src/core/reactor.cc:822
seastar::print_with_backtrace(char const*, bool) at ./build/release/seastar/./build/release/seastar/./seastar/src/core/reactor.cc:867
 (inlined by) seastar::sigsegv_action() at ./build/release/seastar/./build/release/seastar/./seastar/src/core/reactor.cc:3954
 (inlined by) seastar::install_oneshot_signal_handler<11, (void (*)())(&seastar::sigsegv_action)>()::{lambda(int, siginfo_t*, void*)#1}::operator()(int, siginfo_t*, void*) const at ./build/release/seastar/./build/release/seastar/./seastar/src/core/reactor.cc:3936
 (inlined by) seastar::install_oneshot_signal_handler<11, (void (*)())(&seastar::sigsegv_action)>()::{lambda(int, siginfo_t*, void*)#1}::__invoke(int, siginfo_t*, void*) at ./build/release/seastar/./build/release/seastar/./seastar/src/core/reactor.cc:3931
/data/scylla-s3-reloc.cache/by-build-id/57e6a907e0a03a185c4145bb8c7d6316b519b14c/extracted/scylla/libreloc/libc.so.6: ELF 64-bit LSB shared object, x86-64, version 1 (GNU/Linux), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, BuildID[sha1]=8f53abaad945a669f2bdcd25f471d80e077568ef, for GNU/Linux 3.2.0, not stripped

__GI___sigaction at :?
locator::topology::find_node(utils::tagged_uuid<locator::host_id_tag>) const at ././locator/topology.cc:431
locator::token_metadata::get_endpoint_for_host_id_if_known(utils::tagged_uuid<locator::host_id_tag>) const at ././locator/token_metadata.cc:553
db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0::operator()(std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > >&&) const::{lambda(std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > const&)#1}::operator()(std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > const&) const at ././db/view/view.cc:3148
 (inlined by) bool boost::algorithm::any_of<std::__detail::_Node_const_iterator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> >, false, false>, db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0::operator()(std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > >&&) const::{lambda(std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > const&)#1}>(std::__detail::_Node_const_iterator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> >, false, false>, std::__detail::_Node_const_iterator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> >, false, false>, db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0::operator()(std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > >&&) const::{lambda(std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > const&)#1}) at /usr/include/boost/algorithm/cxx11/any_of.hpp:35
 (inlined by) bool boost::algorithm::any_of<std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > >, db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0::operator()(std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > >&&) const::{lambda(std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > const&)#1}>(std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > > const&, db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0::operator()(std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > >&&) const::{lambda(std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > const&)#1}) at /usr/include/boost/algorithm/cxx11/any_of.hpp:50
 (inlined by) db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0::operator()(std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > >&&) const at ././db/view/view.cc:3146
 (inlined by) bool std::__invoke_impl<bool, db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&, std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > > >(std::__invoke_other, db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&, std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > >&&) at /usr/lib/gcc/x86_64-redhat-linux/14/../../../../include/c++/14/bits/invoke.h:61
 (inlined by) std::__invoke_result<db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&, std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > > >::type std::__invoke<db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&, std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > > >(db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&, std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > >&&) at /usr/lib/gcc/x86_64-redhat-linux/14/../../../../include/c++/14/bits/invoke.h:96
 (inlined by) std::invoke_result<db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&, std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > > >::type std::invoke<db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&, std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > > >(db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&, std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > >&&) at /usr/lib/gcc/x86_64-redhat-linux/14/../../../../include/c++/14/functional:120
 (inlined by) auto seastar::internal::future_invoke<db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&, std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > > >(db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&, std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > >&&) at ././seastar/include/seastar/core/future.hh:1172
 (inlined by) seastar::future<std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > > >::then_impl_nrvo<db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0, seastar::future<bool> >(db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&&)::{lambda(seastar::internal::promise_base_with_type<bool>&&, db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&, seastar::future_state<std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > > >&&)#1}::operator()(seastar::internal::promise_base_with_type<bool>&&, db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&, seastar::future_state<std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > > >&&) const::{lambda()#1}::operator()() const at ././seastar/include/seastar/core/future.hh:1486
 (inlined by) _ZN7seastar8futurizeIbE22satisfy_with_result_ofITkSt9invocableZZNS_6futureISt13unordered_mapIN5utils11tagged_uuidIN7locator11host_id_tagEEENS_13basic_sstringIcjLj15ELb1EEESt4hashIS9_ESt8equal_toIS9_ESaISt4pairIKS9_SB_EEEE14then_impl_nrvoIZN2db4view12view_builder24check_view_build_ongoingERKNS7_14token_metadataERKSB_SU_E3$_0NS3_IbEEEET0_OT_ENKUlONS_8internal22promise_base_with_typeIbEERSV_ONS_12future_stateISK_EEE_clES13_S14_S17_EUlvE_EEvS13_SZ_ at ././seastar/include/seastar/core/future.hh:2019
 (inlined by) seastar::future<std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > > >::then_impl_nrvo<db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0, seastar::future<bool> >(db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&&)::{lambda(seastar::internal::promise_base_with_type<bool>&&, db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&, seastar::future_state<std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > > >&&)#1}::operator()(seastar::internal::promise_base_with_type<bool>&&, db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&, seastar::future_state<std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > > >&&) const at ././seastar/include/seastar/core/future.hh:1482
 (inlined by) seastar::continuation<seastar::internal::promise_base_with_type<bool>, db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0, seastar::future<std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > > >::then_impl_nrvo<db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0, seastar::future<bool> >(db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&&)::{lambda(seastar::internal::promise_base_with_type<bool>&&, db::view::view_builder::check_view_build_ongoing(locator::token_metadata const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&, seastar::basic_sstring<char, unsigned int, 15u, true> const&)::$_0&, seastar::future_state<std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > > >&&)#1}, std::unordered_map<utils::tagged_uuid<locator::host_id_tag>, seastar::basic_sstring<char, unsigned int, 15u, true>, std::hash<utils::tagged_uuid<locator::host_id_tag> >, std::equal_to<utils::tagged_uuid<locator::host_id_tag> >, std::allocator<std::pair<utils::tagged_uuid<locator::host_id_tag> const, seastar::basic_sstring<char, unsigned int, 15u, true> > > > >::run_and_dispose() at ././seastar/include/seastar/core/future.hh:748
seastar::reactor::run_tasks(seastar::reactor::task_queue&) at ./build/release/seastar/./build/release/seastar/./seastar/src/core/reactor.cc:2577
seastar::reactor::run_some_tasks() at ./build/release/seastar/./build/release/seastar/./seastar/src/core/reactor.cc:3043
seastar::reactor::do_run() at ./build/release/seastar/./build/release/seastar/./seastar/src/core/reactor.cc:3211
seastar::reactor::run() at ./build/release/seastar/./build/release/seastar/./seastar/src/core/reactor.cc:3101
seastar::app_template::run_deprecated(int, char**, std::function<void ()>&&) at ./build/release/seastar/./build/release/seastar/./seastar/src/core/app-template.cc:276
seastar::app_template::run(int, char**, std::function<seastar::future<int> ()>&&) at ./build/release/seastar/./build/release/seastar/./seastar/src/core/app-template.cc:167
scylla_main(int, char**) at ././main.cc:701
std::function<int (int, char**)>::operator()(int, char**) const at /usr/lib/gcc/x86_64-redhat-linux/14/../../../../include/c++/14/bits/std_function.h:591
main at ././main.cc:2283
__libc_start_call_main at ??:?
__libc_start_main_alias_2 at :?
_start at ??:?

```

**Comment by piodul** [comment]
The freeze from the issue description looks like https://github.com/scylladb/scylladb/issues/20885 - same symptoms, so I think this is a duplicate.

I'm not sure about the segfault, I'll take a look at the backtrace.

**Comment by piodul** [comment]
I think I know what's wrong - looks like an object lifetime issue. It's a different problem so I'll open a new issue and will close this one as a duplicate.

**Comment by piodul** [comment]
Opened https://github.com/scylladb/scylladb/issues/21164.

Closing this issue as a duplicate of https://github.com/scylladb/scylladb/issues/20885.
